app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,74 @@
+class MessagesController < ApplicationController
+  before_action :set_message, only: %i[show edit update destroy]
+
+  def random_greeting
+    message = Message.order('RANDOM()').first
+    render json: { greeting: message.greeting }
+  end
+
+  # GET /messages or /messages.json
+  def index
```

Comment:
Required
- Please keep in mind that starting from the index method all the way down to the destroy method isn't necessary for this milestone and they're not needed for displaying random messages. The only needed one is the random greeting method.